### PR TITLE
[RM Glossary] Cloud User, Tenant, Multi-tenancy

### PR DESCRIPTION
**Pull request by pgoyal01**
[RM Glossary] Added Cloud User (ITU), Tenant (ITU; made existing a Note), and Multi-tenancy (ETSI) per discussion in Issue #666.

### CODE REVIEW

File: doc/ref_model/chapters/glossary.md [hunk 1]
```diff
@@ -64,8 +64,11 @@ To help guide the reader, this glossary provides an introduction to the terminol
 <a name="11.1.3"></a>
 ### 11.1.3 Operational and Administrative Terminology
 
-- **Tenant**: Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
+- **cloud service user**: Natural person, or entity acting on their behalf, associated with a cloud service customer that uses cloud services.
+- **Tenant**: One or more cloud service users sharing access to a set of physical and virtual resources ([ITU](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-Y.3500-201408-I!!PDF-E&type=items)).
+>_*Note*_ Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
 - **Tenant (Internal) Networks**: Virtual networks that are internal to tenant instances.
+- **multi-tenancy**: feature where physical, virtual or service resources are allocated in such a way that multiple tenants and their computations and data are isolated from and inaccessible by each another. ([ETSI}(https://www.etsi.org/deliver/etsi_gs/NFV/001_099/003/01.04.01_60/gs_nfv003v010401p.pdf))
```
Comment (msidana) [inline doc/ref_model/chapters/glossary.md]: Closing needs to be done with square bracket ```]``` for ETSI.

Comment (pgoyal01) [reply doc/ref_model/chapters/glossary.md]: @msidana Done - Thanks

File: doc/ref_model/chapters/glossary.md [hunk 2]
```diff
@@ -64,8 +64,11 @@ To help guide the reader, this glossary provides an introduction to the terminol
 <a name="11.1.3"></a>
 ### 11.1.3 Operational and Administrative Terminology
 
-- **Tenant**: Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
+- **cloud service user**: Natural person, or entity acting on their behalf, associated with a cloud service customer that uses cloud services.
```
Comment (tomkivlin) [inline doc/ref_model/chapters/glossary.md]: Can we add "Examples of such entities include devices and applications." to this line, just to call this point out?

File: doc/ref_model/chapters/glossary.md [hunk 3]
```diff
@@ -64,8 +64,11 @@ To help guide the reader, this glossary provides an introduction to the terminol
...
+- **Tenant**: One or more cloud service users sharing access to a set of physical and virtual resources ([ITU](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-Y.3500-201408-I!!PDF-E&type=items)).
+>_*Note*_ Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.
 - **Tenant (Internal) Networks**: Virtual networks that are internal to tenant instances.
+- **multi-tenancy**: feature where physical, virtual or service resources are allocated in such a way that multiple tenants and their computations and data are isolated from and inaccessible by each another. ([ETSI](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/003/01.04.01_60/gs_nfv003v010401p.pdf))
```
Comment (tomkivlin) [inline doc/ref_model/chapters/glossary.md]: Can we capitalise multi > Multi?

File: doc/ref_model/chapters/glossary.md (hunk 2 above)
Comment (tomkivlin) [inline doc/ref_model/chapters/glossary.md]: Can we capitalise cloud > Cloud?

Comment (pgoyal01) [reply doc/ref_model/chapters/glossary.md]: @tomkivlin Added a note. Thanks

File: doc/ref_model/chapters/glossary.md (hunk 3 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/glossary.md]: @tomkivlin Done - Thanks.

File: doc/ref_model/chapters/glossary.md (hunk 2 above)
Comment (pgoyal01) [reply doc/ref_model/chapters/glossary.md]: @tomkivlin Done - Thanks.

**Pull request merged**